dist/docker/debian/build_docker.sh

```diff
@@ -102,6 +102,10 @@ bconfig --entrypoint  '["/docker-entrypoint.py"]'
 bconfig --cmd  ''
 bconfig --port 10000 --port 9042 --port 9160 --port 9180 --port 7000 --port 7001 --port 22
 bconfig --volume "/var/lib/scylla"
+bconfig --label org.opencontainers.image.ref.name
+bconfig --label org.opencontainers.image.version
+bconfig --label description="ScyllaDB Open Source"
```

Comment:
I expect the description to be different than the summary.
Example for a description:
```
NoSQL data store using the seastar framework, compatible with Apache Cassandra```

Reply:
Changed, for enterprise we have different as @tzach asked. should we add both?